plugins/woocommerce/src/Admin/API/Options.php

```diff
@@ -3,8 +3,19 @@
  * REST API Options Controller
  *
  * Handles requests to get and update options in the wp_options table.
+ *
+ * IMPORTANT: This API is for legacy support only. DO NOT add new options here.
```

Comment:
I think it'll be great to add some information on why this was deprecated as well. p90Yrv-2vK-p2#comment-6482

Reply:
Good idea! Added in b62573173a01ce87bb074d215d06e6197301ef67.